cmd/geth/dbcmd.go

```diff
@@ -80,48 +78,29 @@ Remove blockchain and state databases`,
 		Action:    utils.MigrateFlags(inspect),
 		Name:      "inspect",
 		ArgsUsage: "<prefix> <start>",
-		Flags: []cli.Flag{
-			utils.DataDirFlag,
-			utils.AncientFlag,
+		Flags: append([]cli.Flag{
 			utils.SyncModeFlag,
-			utils.MainnetFlag,
-			utils.RopstenFlag,
-			utils.SepoliaFlag,
-			utils.RinkebyFlag,
-			utils.GoerliFlag,
-		},
+		}, utils.GroupFlags(utils.NetworkFlags, utils.DatabasePathFlags)...),
 		Usage:       "Inspect the storage size for each type of data in the database",
 		Description: `This commands iterates the entire database. If the optional 'prefix' and 'start' arguments are provided, then the iteration is limited to the given subset of data.`,
 	}
 	dbStatCmd = cli.Command{
 		Action: utils.MigrateFlags(dbStats),
 		Name:   "stats",
 		Usage:  "Print leveldb statistics",
-		Flags: []cli.Flag{
-			utils.DataDirFlag,
+		Flags: append([]cli.Flag{
 			utils.SyncModeFlag,
-			utils.MainnetFlag,
-			utils.RopstenFlag,
-			utils.SepoliaFlag,
-			utils.RinkebyFlag,
-			utils.GoerliFlag,
-		},
+		}, utils.GroupFlags(utils.NetworkFlags, utils.DatabasePathFlags)...),
```

Comment:
Nitpick (kind of in general). 
I wonder if it wouldn't become cleaner if you used utils.GroupFlags as the outer wrapper, instead of `append`. 

In this case, it would be 

```
		Flags: utils.GroupFlags([]cli.Flag{utils.SyncModeFlag},
			utils.NetworkFlags,
			utils.DatabasePathFlags),
```